src/styles/main.scss

```diff
@@ -0,0 +1,72 @@
+$clock-size: 200px;
+$center-size: 10px;
+
+$minute-arrow-color: burlywood;
+$second-arrow-color: aliceblue;
+
+$second-arrow-height: 90px;
+$minute-arrow-height: 70px;
+$arrow-width: 6px;
+
+@mixin centerSize($size) {
+  width: $size;
+  height: $size;
+}
+
+.clock {
+  margin: 100px auto;
+  position: relative;
+
+  width: $clock-size;
+  height: $clock-size;
+  background-color: darkgray;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    top: (($clock-size / 2) - ($center-size / 2));
+    left: (($clock-size / 2) - ($center-size / 2));
+
+    @include centerSize($center-size);
+    background-color: burlywood;
+    border-radius: 50%;
+  }
+
+  &__second-arrow {
+    position: absolute;
+    top: $clock-size / 2 - $second-arrow-height;
+    left: ($clock-size / 2 - $arrow-width / 2);
+
+    width: $arrow-width;
+    height: $second-arrow-height;
+
+    background-color: $second-arrow-color;
+    border-radius: 50%;
+    transform-origin: bottom center;
+    animation: rotate 60s linear infinite;
+  }
+
+  &__minute-arrow {
+    position: absolute;
+    top: $clock-size / 2 - $minute-arrow-height;
+    left: ($clock-size - $arrow-width) / 2;
```

Comment:
you use weird formulas repeat this code in three places with minor changes.
put it in extends
```
    position: absolute;
    left: 50%;
    bottom: 50%;

    transform-origin: bottom center;
```
and 
```
    transform: translate(-50%, 50%);
```

for clock__center